cinema/views.py

```diff
@@ -0,0 +1,40 @@
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+from cinema.serializers import CinemaSerializer
+
+from cinema.models import Movie
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    movies = Movie.objects.all()
+    if request.method == "GET":
+        serializer = CinemaSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = CinemaSerializer(movies, data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Use `serializer.is_valid(raise_exception=True)` to simplify code. When you use raise_exception=True, if the serializer is invalid, an exception (serializers.ValidationError) is raised. DRF catches this exception and automatically returns a 400 Bad Request response with the appropriate error messages.
```suggestion
        serializer.is_valid(raise_exception=True)
        serializer.save()
        return Response(serializer.data, status=status.HTTP_201_CREATED)
```